### PR TITLE
Translation-ja: Add contributor/ja/02-becoming-a-contributor-article-ja

### PR DESCRIPTION
**Pull request by ystk**
This PR adds a translation for 02-becoming-a-contributor-article.

### CODE REVIEW

**Comment by rrrutledge** [comment]
Tagging @johtani for potential review?

**Review by johtani — APPROVED** [review]
Basically LGTM. I left only one small comment :D

File: contributor/ja/02-becoming-a-contributor-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+== InnerSourceコントリビューターになる
+
+InnerSourceのコントリビューターは、通常のチームの境界外で活動し、組織のサイロを横断するリンクになります。
+そのため、彼らはこの作業をより効果あるものにするための、いくつかの共通の方法を意識する必要があります。
+
+=== マインドセットの共有
+
+それでは - あなたはチームの製品に、新しい機能を実装しています。
+この機能を動作させるためには、いくつかの機能が必要です。
+実装に直ぐに加わる代わりに、少し落ち着いて考えます：この機能は一般的な課題なのだろうか？
```
[the remark this answers is not in the document]
Comment (ystk) [reply contributor/ja/02-becoming-a-contributor-article-ja.asciidoc]: ありがとうございます。提案いただいた修正を取り込みました。

**Comment by rrrutledge** [comment]
Awesome!  @johtani if it looks good to you then I am happy to merge.

**Review by johtani — APPROVED** [review]
LGTM!

**Review by rrrutledge — APPROVED** [review]
Great!

**Pull request merged**